packages/components/organisms/f-takeawaypay-activation/src/components/ActivationNotLoggedIn.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div>
+        <bag-celebrate-bg-icon :class="$style['c-takeawaypayActivation-icon']" />
+
+        <h1 :class="$style['c-takeawaypayActivation-title']">
+            {{ $t('messages.titleActivationNotLoggedIn') }}
+        </h1>
+
+        <p>
+            {{ $t('messages.descriptionActivationNotLoggedIn') }}
+        </p>
+
+        <div :class="$style['c-takeawaypayActivation-actions']">
+            <f-button
+                :href="loginUrl"
+                data-test-id="takeawaypay-activation-not-loggedin-login-link"
+                button-type="primary"
+                button-size="large"
+                is-full-width>
+                {{ $t('actions.loginToAccount2') }}
+            </f-button>
+
+            <f-button
+                :href="registrationUrl"
+                data-test-id="takeawaypay-activation-not-loggedin-registration-link"
+                button-type="outline"
+                button-size="large"
+                is-full-width>
+                {{ $t('actions.createAccount') }}
+            </f-button>
+        </div>
+    </div>
+</template>
+
+<script>
+import FButton from '@justeat/f-button';
+import '@justeat/f-button/dist/f-button.css';
+import { BagCelebrateBgIcon } from '@justeat/f-vue-icons';
+
+export default {
+
+    components: {
+        FButton,
+        BagCelebrateBgIcon
+    },
+
+    props: {
+        loginUrl: {
+            type: String,
+            required: true
+        },
+        registrationUrl: {
+            type: String,
+            required: true
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+
+.c-takeawaypayActivation-icon {
```

Comment:
As these styles look like they are the same across multiple views/components, you can pull them out into a separate scss file and import them into each file to avoid having to repeat them in each component view.

Reply:
Yeah, makes sense. Had similar thoughts, but wasn't sure if it good approach. Will fix.